### ISSUE TITLE
 Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by kpietrzak-web** [Manweill/swagger-axios-codegen#20]
Error appears in swagger-axios-codegen/dist/definitionCodegen/createDefinitionClass.js:19:39

when:
if in swagger output I have definition without properties. If definition contains even empty properties it works

example:
```
"definitions": {
    // triggers error
    "ItemId": {
      "type": "object"
    },
    // works
    "ItemId": {
      "type": "object",
      "properties": {}
    },
}

sulution
change:
```
    const propertiesEntities = Object.entries(properties);
```
to:
```
    const propertiesEntities = Object.entries(properties || {});
```

or check if properies are undefines then don't even try to get entries.

**Comment by Manweill** [comment]
@kpietrzak-web 
Wecome PR.

**Comment by Manweill** [comment]
fix in v0.5.3
